app/controllers/api/physical_servers_controller.rb

```diff
@@ -65,6 +65,10 @@ def apply_config_pattern_resource(type, id, data)
       action_result(false, err.to_s)
     end
 
+    def firmware_binaries_query_resource(object)
```

Comment:
Hey @miha-plesko 

we used to put sub collections related methods to the `Api::Subcollections::NameOfCollection` module.
In your case probably something like `Api::Subcollections::FirmwareBinaries`. See `Subcollections::EventStreams` in this controller class as example.

Does make sense to put it here for you ?
thanks!

Reply:
Thanks, yes it makes sense. My "change as little as possible" was a bit too extream. Fixed